stable/magento/templates/externaldb-secrets.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if not .Values.mariadb.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-%s" .Release.Name "externaldb"  }}
```

Comment:
I think we should be using the `magento.fullname` helper template here, `{{ template "magento.mariadb.fullname" . }}-externaldb` maybe??

Reply:
The database does not have to be MariaDB so I don't think it is necessary

Reply:
oh sorry.. i mean to suggest `{{ template "magento.fullname" . }}-externaldb`

Reply:
That makes more sense. Done!